webpack/assets/javascripts/react_app/components/notifications/notifications.test.js

```diff
@@ -137,17 +137,4 @@ describe('notifications', () => {
     wrapper.find(closeButtonSelector).simulate('click');
     expect(toJson(wrapper)).toMatchSnapshot();
   });
-
-  it('should close the notification box when click outside of the box', () => {
```

Comment:
Removed this test because it is not doing the job.
Replaced it with integration test.